tools/log.c

```diff
@@ -62,19 +62,10 @@ _printf_format_(6, 0) static void log_kmod(void *data, int priority, const char
 		return;
 
 	if (log_use_syslog) {
-#ifdef ENABLE_DEBUG
 		syslog(priority, "%s: %s:%d %s() %s", prioname, file, line, fn, str);
-#else
-		syslog(priority, "%s: %s", prioname, str);
```

Comment:
wat? this doesn't match the commit message. Now everybody is logging the file, line, function name.

Reply:
Humn... tools that want a more sophisticate log are supposed to override the ctx->log_fn via kmod_set_log_fn().

But yes. If this is only about the priority name, I'm fine with it.